lib/temporal/workflow/context.rb

```diff
@@ -304,6 +304,41 @@ def cancel(target, cancelation_id)
         end
       end
 
+      # Send a signal from one workflow to another workflow. For signaling from the client,
+      # see Client#signal_workflow.
+      #
+      # @param workflow [Temporal::Workflow, String] workflow class or name. When a workflow class
+      #   is passed, its config (namespace, task_queue, timeouts, etc) will be used
+      # @param signal_name [String] name of signal
+      # @param input [String, Array, nil] optional arguments for the signal
+      # @param args [Hash] keyword arguments to be pass to workflow's #execute method
```

Comment:
This argument is missing from the method signature

Reply:
Fixed.